test/spec/modules/rubiconBidAdapter_spec.js

```diff
@@ -1155,8 +1155,8 @@ describe('the rubicon adapter', function () {
             .that.deep.equals(['a', 'b', 'c']);
 
           expect(slot).to.have.property('visitor').that.is.an('object');
-          expect(slot.visitor).to.have.property('ucat').that.equals('new');
-          expect(slot.visitor).to.have.property('lastsearch').that.equals('iphone');
+          expect(slot.visitor).to.have.property('ucat').that.deep.equals(['new']);
+          expect(slot.visitor).to.have.property('lastsearch').that.deep.equals(['iphone']);
```

Comment:
Now instead of sending a bad request it is sending the array wrapped data which will be accepted by Frank.